src/lib/Scenes/Artwork/Components/ArtworkExtraLinks.tsx

```diff
@@ -62,8 +62,8 @@ export class ArtworkExtraLinks extends React.Component<ArtworkExtraLinksProps> {
             Have a question?{" "}
             <Text style={{ textDecorationLine: "underline" }} onPress={() => this.handleReadOurAuctionFAQsTap()}>
               Read our auction FAQs
-            </Text>
-            . Or{" "}
+            </Text>{" "}
+            or{" "}
```

Comment:
didn't we just change this the other day? is this an accidental revert?

Reply:
Yes but it was incorrect before. This is a fix.